libs/image/include/image/ColorSpace.h

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IMAGE_COLORSPACE_H_
+#define IMAGE_COLORSPACE_H_
+
+#include <image/Image.h>
+
+#include <math/scalar.h>
+#include <math/vec3.h>
+#include <math/vec4.h>
+
+#include <memory>
+
+namespace image {
+
+template <typename T>
+inline math::float4 linearToRGBM(const T& linear) {
```

Comment:
This is not a color space conversion. We should maybe name the file differently.